components/brave_vpn/brave_vpn_service_desktop.cc

```diff
@@ -256,5 +257,11 @@ void BraveVpnServiceDesktop::GetAllRegions(GetAllRegionsCallback callback) {
   for (const auto& region : regions_) {
     regions.push_back(region.Clone());
   }
+
+  std::sort(regions.begin(), regions.end(),
```

Comment:
How about doing this sorting in `ParseAndCacheRegionList()`?
Then, we don't need to sort whenever `GetAllRegions()` is called.